terraform/aws/modules/aws_vpc/outputs.tf

```diff
@@ -13,3 +13,7 @@ output "ipv4_cidr_block" {
 output "ipv6_cidr_block" {
   value = "${cidrsubnet(aws_vpc.default.ipv6_cidr_block, 8, 1)}"
 }
+
+output "region" {
```

Comment:
I found this output useful - feel free to delete it.

Reply:
I also get TF output running `make create-vpc` but not `make apply`. Am I doing something wrong?

Reply:
nah you aren't doing anything wrong

I'm sure there are a bazillion things that I should have done better.  My primary focus was trying to enable a developer to easily spin up ALL scenarios concurrently which changed what I thought should be output to the console for each scenario.

I'm totally open to suggestions and whatnot, but note that you have access to the buildkite pipeline version of this stuff which is even easier to launch the full matrix of tests without any setup of your workstation.